MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,209 @@
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public ContactInfo ContactDetails { get; set; }
+
+        public IList<string> CountryNames { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class ContactInfo
+    {
+        public string Name { get; set; }
+
+        public string Password { get; set; }
+
+        public string Street { get; set; }
+
+        public string City { get; set; }
+
+        public string ZipCode { get; set; }
+
+        public string Country { get; set; }
+
+        public string Email { get; set; }
+    }
+
+{% endhighlight %}
+
+## Dynamically add DataForm items
+You can dynamically add the items using `Items` property of `SfDataForm`.
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.Add(new DataFormTextItem() { FieldName = "State" });
+    dataForm.Items.Add(new DataFormTextItem() {FieldName = "Country" });
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically remove manually added dataform items
+You can dynamically remove the `DataFormViewItem` from the `Items` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.RemoveAt(2);
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically clear manually added dataform items
+You can dynamically clear all the item views from `SfDataForm`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.Clear();
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically reset manual dataform items
+You can reset the `SfDataForm.Items` and we can reset two or more items at a time.
+
+{% tabs %}
+{% highlight C# %}
+
+    DataFormTextItem textItem = new DataFormTextItem() { FieldName = "Age" };
+    dataForm.Items[2] = textItem;
+    
+    
+    DataFormDateItem dateItem = new DataFormDateItem() { FieldName ="Date Of Birth", RowOrder = 2, ItemsOrderInRow = 1, ColumnSpan = 3 });
+
+    dataForm.Items[3] = dateItem;
+    
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically add Dataform group items
+You can dynamically add custom group items using `Items` property of `SfDataForm`.
+{% tabs %}
+{% highlight C# %}
+
+    DataFormGroupItem dataFormGroupItem = new DataFormGroupItem();
+    dataFormGroupItem.Name = "Name";
+    dataFormGroupItem.Items = new ObservableCollection<DataFormItem>();
+
+    dataFormGroupItem.Items.Add(new DataFormTextItem() { FieldName = "First Name" });
+    dataFormGroupItem.Items.Add(new DataFormTextItem() { FieldName = "Middle Name" });
+    dataFormGroupItem.Items.Add(new DataFormTextItem() { FieldName = "Last Name" });
+
+    dataForm.Items.Add(dataFormGroupItem);
+    
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Adding custom editor view using Manual DataFormItem
+
+You can directly set a custom view as an `EditorView` of `DataFormCustomItem`. In this case, the editor view will not be generated based on field type and `RegistorEditor`.
+{% tabs %}
```

Comment:
when AutoGenerateItems is false.

You can add custom editor manually by adding the custom view as an `EditorView` of `DataFormCustomItem`. In this case, the editor view will not be generated based on field type and `RegistorEditor`. 

(add refrence of UG RegistorEditor custom editor)

Reply:
content updated